vignettes/basic-principles.Rmd

```diff
@@ -14,11 +15,25 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r setup, results = 'hide'}
+```{r setup, echo = FALSE, results = 'hide'}
```

Comment:
Looking at vignettes from other packages, they (almost) always start with a block that contains the `library` call to the package

Some examples:
vignette("sql", package = "dbplyr") 
vignette("DBI", package = "DBI")

`usethis::use_vignette()` also creates that block. 

Consider splitting the library call out from this block and leave it explicit to the users.

Reply:
Good call; I've added a visible `library(SCDB)` call to the vignette. The library is still loaded in the setup block to prepare the `example_data` object used in the vignette.